### DISCUSSION TITLE
Enocoutered Error in runing get_cube.py

### DISCUSSION THREAD

**Discussion by suryaphy520** [FLOSIC/PublicRelease_2020#31] [Q&A] [answered]
Respected sir, When I run the get_cube.py file after all necessary steps it shows an error, read as ValueError: max() arg is an empty sequence. How can I reslove this issue. Can you plz help me.

**Comment by yohyamamoto** [comment, marked as answer]
It's most likely your arguments for the get_cube.py.

First compile the cube_exe by running `make cube -f Makefile `

With the current code, I had to edit the Makefile where the line starts with CUBEOBJ needs to be: 

`CUBEOBJ = mod_periodic_table.o new_input_reader.o $(CUBEBASICS) $(FBASICS) $(METAGGA) $(EXTRAFIL) $(POSTACTIVE) $(BASIS) $(DISP) $(WFX) $(DMAT) $(GROUPOBJ) $(EFPOBJ) $(SIC) $(SIC2) $(SIC3) $(LOCALSIC) $(LEVY_PERDEW)`

This change is pushed for your convenience.

Once you have cube_exe, you can use the get_cube.py with a syntax as follows,

`python3  get_cube.py --cube cubefilename calculation_directory_path cube_exe_directory_path`

Say you have the directory tree something like

```
flosic_dir
├── cube_exe
├── get_cube.py
├── nrlmol_exe
└── test_calculation
    └── generate_cube_here
```

Then you go to the test_calculation and run
`python3 ../get_cube.py --cube cubefilename  ./ ../`


I suppose you can also copy these files to your working directory. In which case, `python3 get_cube.py --cube cubefilename  ./ ./`
